<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="INSGH045PMS0" ignored="0" name="Unix Extract NBA Channel File Regression Test" tests="1" failures="1" timestamp="2021-01-12T12:50:51 IST" time="1.027" errors="0">
  <testcase name="VerifyExtracDataLoadedSuccessfully" time="1.027" classname="Mccm.Unix.Outbound.App_Unix_Outbound_Extract_Test">
    <failure type="com.jcraft.jsch.JSchException" message="java.net.ConnectException: Connection refused: connect">
      <![CDATA[com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
at com.jcraft.jsch.Util.createSocket(Util.java:349)
at com.jcraft.jsch.Session.connect(Session.java:215)
at com.jcraft.jsch.Session.connect(Session.java:183)
at Mccm.Unix.Outbound.App_Unix_Outbound_Extract_Test.VerifyExtracDataLoadedSuccessfully(App_Unix_Outbound_Extract_Test.java:82)
at java.util.ArrayList.forEach(Unknown Source)
at TestRuner.MccmTestRunner.main(MccmTestRunner.java:30)
Caused by: java.net.ConnectException: Connection refused: connect
at java.net.DualStackPlainSocketImpl.connect0(Native Method)
at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
at java.net.PlainSocketImpl.connect(Unknown Source)
at java.net.SocksSocketImpl.connect(Unknown Source)
at java.net.Socket.connect(Unknown Source)
at java.net.Socket.connect(Unknown Source)
at java.net.Socket.<init>(Unknown Source)
at java.net.Socket.<init>(Unknown Source)
at com.jcraft.jsch.Util.createSocket(Util.java:343)
... 29 more
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- VerifyExtracDataLoadedSuccessfully -->
</testsuite> <!-- Unix Extract NBA Channel File Regression Test -->
